map_data/geographical_region.txt

```diff
@@ -767,24 +767,52 @@ world_pandaria={
 custom_deserts = {
 	regions = {
 		world_kalimdor_desolace
-		world_kalimdor_thousand_needles
 		world_kalimdor_uldum
 		world_kalimdor_silithus
 		world_kalimdor_tanaris
 		world_south_seas_zandalar_voldun
-		world_south_seas_kezan
 	}
 }
+custom_drylands = {
+	regions = {
+		world_kalimdor_thousand_needles
+		world_kalimdor_barrens
+		world_kalimdor_durotar
+	}
+	duchies = {
+		d_edj
+		d_undermine
+
+		d_badland
+		d_lethlor
+
+	}
+}
+
+custom_arid_regions = {
+	regions = {
+		custom_deserts
+		custom_drylands
+	}
+		
+}
 
 custom_south_seas_except_deserts = {
 	regions = {
 		world_south_seas_plunder_isle
 		world_south_seas_zandalar_except_voldun
 		world_south_seas_isle_of_giants
 		world_south_seas_tel_abim
-		world_south_seas_darkspear_islands
 		world_south_seas_vashjir
 	}
+
+	duchies = {
+		#Darkspear Islands#
+		d_darkspear_islands
+		d_paradise_islands
+		d_turtle_rock
+		d_verdant_wilds
+	}
```

Comment:
Don't get the change.
The deleted region includes every duchy you mentioned.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/115126983-3a099b00-9fe4-11eb-832b-9f3b17e11045.png)

</details>

Reply:
I removed turtle rock because there are no jungles, but we agreed to add them in anyway. Will revert back to region.